examples/cifar10/README.md

```diff
@@ -1,142 +1,52 @@
-    Define the steps used for a stepped learning rate the steps are specified
-    using Python syntax; e.g. to drop the LR by a factor of 0.1 at epochs 60,
-    120 and 160 use:
-    `--lr_sched_steps="[[60, 0.1], [120, 0.01], [160, 0.001]]"`
-- `--num_epochs` (default=200): the number of epochs to train for
-- `--l2_reg` (default=0.005): the amount of L2 regularization to apply
-    (on all parameters)
-- `--batch_size` (default=256): mini-batch size
-- `--arch` (default=wrn26_10): network architecture
-  -  `wrn26_10`: Wide ResNet, 26 layers, 10x width
-  -  `wrn26_6_ss`: Wide ResNeXt, 26 2x96d with shake-shake regularization
-  -  `pyramid`: PyramidNet, pyramid alpha=200, 272 layers, ShakeDrop
-        regularization
-- `--wrn_dropout_rate` (default=0.3): DropOut rate used in Wide ResNet
-    (on all residual blocks)
-- `--rng` (default=0): Random seed used for network initialization and
-    stochasticity during training
-
-
-## Issues
-
-We apply L2 regularization to weights and biases, whereas it is mostly *not*
-applied to biases.
-
-
-## References
-
-Various code was used as a reference to help replicate baseline results.
-In particular the Fast AutoAugment codebase was particularly helpful
-as its config files describe training regimes for the wide ResNet
-with shake-shake and PyramidNet baselines.
-
-Shake-shake author's implementation:
-https://github.com/xgastaldi/shake-shake
-
-PyTorch implementation of shake-shake:
-https://github.com/owruby/shake-shake_pytorch
-
-Fast AutoAugment that uses a number of different models:
-https://github.com/kakaobrain/fast-autoaugment
+## CIFAR10 classification
+Trains a **family** ResNet-style models (He *et al.*, 2015; Zagoruyko and Komodakis, 2017; Han *et al.*, 2017) for the CIFAR10 classification task (Krizhevsky, 2009).
+
+This example implements several architectures, regularization methods (Gastaldi, 2017; Yamada *et al.*, 2018) and learning rate schedules that can be used in various combinations.
+
+### Requirements
+* TensorFlow dataset `cifar10`
+
+### Supported setups
+The model should run with other configurations and hardware, but explicitly tested on the following.
+
+#### Wide ResNet: 26 layers, 10x width (Zagoruyko and Komodakis, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| --- | --- | --- | --- | --- | --- |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 4h 36m | 4.45% | [2020-03-22](https://tensorboard.dev/experiment/1kE2bq9RR7WrT7Zw14fNpg/) |
+| 8 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 57m | 3.93% | [2020-03-22](https://tensorboard.dev/experiment/IpzU0txnR7uZnExGmk7IEA/) |
+
+#### Wide ResNet: 26 layers, 6x width, Shake-Shake regularization (Gastaldi, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 3h 38m | 3.43% | [2020-03-22](https://tensorboard.dev/experiment/dJc9e4k1R5mC0DPVZyYrpg/) |
```

Comment:
These don't render as tables for some reason (also they don't have the row with --- | ---, maybe that's related?)